cmd/xl-v1-multipart.go

```diff
@@ -315,13 +321,14 @@ func (xl xlObjects) PutObjectPart(ctx context.Context, bucket, object, uploadID
 	}
 
 	// Validates if upload ID exists.
-	if !xl.isUploadIDExists(bucket, object, uploadID) {
+	if !xl.isUploadIDExists(ctx, bucket, object, uploadID) {
 		preUploadIDLock.RUnlock()
-		return pi, errors.Trace(InvalidUploadID{UploadID: uploadID})
+		logger.LogIf(ctx, InvalidUploadID{UploadID: uploadID})
```

Comment:
No need to log here since `isUploadIDExists` already takes care of the `InvalidUploadID` error condition.

Reply:
done